src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -3334,8 +3344,13 @@ namespace winrt::Microsoft::Terminal::Control::implementation
 
         _lastHoveredCell = terminalPosition;
 
+        uint16_t newId{ 0u };
+        // we can't use auto here because we're pre-declaring newInterval.

```

Comment:
thank you, I would not have been able to decipher that next line without this comment